db/migrate/20210221211809_adds_category_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddsCategoryToProducts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :products, :category, :string
```

Comment:
We have not showed you this so this is good for this iteration, but in the future when you want an attribute to be a collection of set values, look into using an `enum`